### DISCUSSION TITLE
1.1

### DISCUSSION THREAD

**Discussion by benfulton** [hahnlab/CAFE5#128] [Announcements]
### Added
- The output file Base_report.cafe or Gamma_report.cafe has been added. This file has a 
format roughly the same as the CAFE 4 output, so tools and scripts that work with this
format should run properly.
- The tutorial now includes a cafe5_draw_tree.py script, demonstrating how to use BioPython
to create a tree.

### Changed
- Ordering of nodes has changed. The order should now be the same, relative to the source Newick
tree, as in the R package Ape.
- The model_change.tab file no longer includes a "+" sign before positive change counts.
- The simulator handles different max family and max root sizes correctly.
- The DocTest unit tester has been updated to v2.4.9.
- The application will no longer attempt to compute P-Values or a reconstruction, if the user
specifies a lambda and the calculated -lnL is infinite. 
- The configure script now checks for the existence of mkl.h before attempting to use it.
- Building the application should no longer fail if OpenMP is missing. (It will probably run
much slower, however)

<hr /><em>This discussion was created from the release <a href='https://github.com/hahnlab/CAFE5/releases/tag/v5.1'>1.1</a>.</em>

**Comment by JiangLab2020** [comment]
[metazoa_78_cafe.txt](https://github.com/hahnlab/CAFE5/files/12377935/metazoa_78_cafe.txt)
[metazoa_78_groupsAllNum.txt](https://github.com/hahnlab/CAFE5/files/12377936/metazoa_78_groupsAllNum.txt)
I run the command line: cafe5 -i metazoa_78_groupsAllNum.txt -t metazoa_78_cafe.txt
no result with error: Failed to initialize any reasonable values

**Reply by benfulton** [reply]
See https://github.com/hahnlab/CAFE5/discussions/132

**Reply by JiangLab2020** [reply]
What is the maximum size of the tree for cafe5？ 
Where can I download 'Count' from ?

**Reply by JiangLab2020** [reply]
[plant_25_cafe.txt](https://github.com/hahnlab/CAFE5/files/12387443/plant_25_cafe.txt)
[plant_25_groupsAllNum.txt](https://github.com/hahnlab/CAFE5/files/12387444/plant_25_groupsAllNum.txt)
cafe5 -i plant_25_groupsAllNum.txt -t plant_25_cafe.txt -o plant_25
In this case, I only used 25 species in the tree, it still has the error.

**Reply by benfulton** [reply]
The Count paper is here: https://academic.oup.com/bioinformatics/article/26/15/1910/189891 

There is no maximum size of the tree, but if your data is too variable then there will be no sensible lambda value.

**Reply by JiangLab2020** [reply]
OK，thanks
